cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField
+    duration = models.IntegerField
```

Comment:
Are you sure that the fields are created in your table? Did you remember to call these descriptors ;)